src/frontend-samples/version-compare-sample/VersionCompareApi.ts

```diff
@@ -0,0 +1,159 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { DbOpcode, Id64Array, Id64String } from "@bentley/bentleyjs-core";
+import { ColorDef, FeatureAppearance } from "@bentley/imodeljs-common";
+import { AuthorizedFrontendRequestContext, EmphasizeElements, FeatureOverrideProvider, FeatureSymbology, IModelApp, NotifyMessageDetails, OutputMessagePriority, Viewport } from "@bentley/imodeljs-frontend";
+import { VersionCompareClient } from "./VersionCompareClient";
+
+export interface NamedVersion {
```

Comment:
Any reason to not use [Version](https://www.itwinjs.org/reference/imodelhub-client/imodelhubclient/version/)?

Reply:
I was unaware it existed.  Thank you for pointing this out.

Reply:
After looking at the type declaration, it requires wsgId, and ecId.  It feels strange to use a type that forces me to assign useless data when the property names in the response don't match those in the type.

Reply:
After adopting the IModelClient.verison.get method to populate the Named Version, using this type makes sense, and I'm now doing so.